src/main/lombok/no/hvl/dat250/h2020/group5/entities/Poll.java

```diff
@@ -25,6 +23,7 @@
     @SequenceGenerator(name="PollID_Sequence", initialValue = LOWEST_4_DIGIT_BASE36)
     @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "PollID_Sequence")
     @Convert(converter = AlphaNumeric2Long.class)
+    @Setter(AccessLevel.NONE)
```

Comment:
Jeg har noen tester som avhenger av å sette Id'en :S

I tillegg bør man visst ha setters hvis man har getter fant jeg it her: https://stackoverflow.com/questions/12369641/is-there-any-reason-to-not-generate-setters-and-getters-for-id-fields-in-jpa

Reply:
Aha, har endret det nå :)